### PR TITLE
Added Babylonian Method in c and c++

### LINKED ISSUE #915: Babylonian method in c and c++
Babylonian method is used for finding the square root of a number.

### PR DESCRIPTION
**Pull request by somya-kapoor**
Fixes #915 
#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [x] Sample Input / Output.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

#### Changes proposed in this pull request:

- Added Babylonian method in c and c++

#### Languages Used:
- C
- C++

#### Files Added:
- Babylonian_Method.c
- Babylonian_Method.cpp

### CODE REVIEW

File: Babylonian_Method/Babylonian_Method.cpp [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    return number; 
+} 
+  
+int main() 
+{ 
+    int num ;
+    cout << "Enter a number :\n" ;
+    cin >> num; 
+    cout << "Square root of " << num << " is " << squareRoot(num); 
+
```
Comment (sridharjajoo) [inline Babylonian_Method/Babylonian_Method.cpp]: remove extra line

Comment (somya-kapoor) [reply Babylonian_Method/Babylonian_Method.cpp]: @sridharjajoo I have done the required changes, please review.

Comment (somya-kapoor) [reply Babylonian_Method/Babylonian_Method.cpp]: @sridharjajoo I have done the required changes, please review.

**Comment by somya-kapoor** [comment]
@sridharjajoo I have done the required changes, please review.

**Pull request merged**